### PR TITLE
adding preliminary info card

### PR DESCRIPTION
**Pull request by kimcharlene**
- implemented basic info card component
- created empty pages (Design, STEM, Teaching)
- minor css changes to navbar

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -1,176 +1,19 @@
...
-              <p style={descriptionStyle}>{link.description}</p>
-            </span>
-          </li>
-        ))}
-      </ul>
-      <img
-        alt="Gatsby G Logo"
-        src="data:image/svg+xml,%3Csvg width='24' height='24' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M12 2a10 10 0 110 20 10 10 0 010-20zm0 2c-3.73 0-6.86 2.55-7.75 6L14 19.75c3.45-.89 6-4.02 6-7.75h-5.25v1.5h3.45a6.37 6.37 0 01-3.89 4.44L6.06 9.69C7 7.31 9.3 5.63 12 5.63c2.13 0 4 1.04 5.18 2.65l1.23-1.06A7.959 7.959 0 0012 4zm-8 8a8 8 0 008 8c.04 0 .09 0-8-8z' fill='%23639'/%3E%3C/svg%3E"
-      />
+    <main>
```
Comment (seongsikk) [inline src/pages/index.js:8]: This seems like a duplicate to "src/pages/teaching.js'?

**Review by seongsikk — COMMENTED** [review]
Looks like a good start overall, can you also please add some screenshots to the description of the pages that you created so far for all of the PRs? (Easier to visually look at the changes you are making). Thanks!

**Comment by kimcharlene** [comment]
pr#1- fig1
Screenshot of bio card highlighted.

**Comment by seongsikk** [comment]
Looks good!

**Pull request merged**